docs/core/config/auth/databases/ldap.md

```diff
@@ -34,35 +34,45 @@ you'll need to create an administrator account that has access to the
 `userPassword` field. With OpenLDAP this can be done by modifying
 `/etc/ldap/slapd.conf`:
 
-```
+::: code-group
+```[dovecot.conf]
 # there should already be something like this in the file:
 access to attribute=userPassword
     by dn="<dovecot's dn>" read  # just add this line
     by anonymous auth
     by self write
     by * none
 ```
+:::
 
-Replace `<dovecot's dn>` with the DN you specified in `dovecot-ldap.conf`'s
-`dn` setting.
+Replace `<dovecot's dn>` with the DN you specified in `ldap_auth_dn`
+in `dovecot.conf`'s ldap settings.
 
 ### Dovecot Configuration
 
 The two important settings in password lookups are:
 
-- `pass_filter` specifies the LDAP filter how user is found from the
+- `ldap_filter` specifies the LDAP filter how user is found from the
```

Comment:
This should be `[[setting,ldap_filter]]` I think, similarly elsewhere where possible

Reply:
should all the settings be moved in settings.js then ?

Reply:
yes